core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -540,19 +540,21 @@ case class ThreadStackTrace(
     lockName: Option[String],
     lockOwnerName: Option[String],
     suspended: Boolean,
-    inNative: Boolean) {
+    inNative: Boolean,
+    isDaemon: Boolean,
+    priority: Int) {
```

Comment:
Has priority ever meant anything for Java threads? I thought it had no effect. Is this useful info?

Reply:
The JVM supports a priority-based scheduling algorithm for thread scheduling. For example, Spark UI threads get a priority of 5, while its acceptors get 3. However, given the limited scope of my knowledge, I can't tell whether it actually works or not.

Reply:
Yeah, practically it does not convey much given how jvm and linux use thread priority in case of java.
Wondering if it might be distracting for users - they cant really do much about it, while it ends up being a red herring while analyzing performance.

I would be fine with dropping it, given the marginal value (if any) they bring.